lib/puppet/provider/scheduled_task/win32_taskscheduler.rb

```diff
@@ -333,6 +343,17 @@ def translate_hash_to_trigger(puppet_trigger)
       self.fail "Unknown schedule type: #{puppet_trigger["schedule"].inspect}"
     end
 
+    if puppet_trigger['minutes_interval']
+      integer_interval = Integer(puppet_trigger['minutes_interval'])
+      self.fail 'minutes_interval must be an integer greater or equal to 0' unless integer_interval >= 0
```

Comment:
Our coding style at puppet has evolved since this code was originally written, and our current preference is to avoid `unless`, especially as this is double-negative logic.

I'd be fine if you left this as is, and added another commit that changed the double-negative logic here and `unless start_date >= Date.new(1753, 1, 1)` as well.